resources/js/components/field-conditions/Condition.vue

```diff
@@ -117,7 +121,7 @@ export default {
 
         fieldOptions() {
             return _(this.suggestableFields)
-                .reject(field => field.handle === this.config.handle || this.condition.field === field.handle)
+                .filter(field => this.conditions.filter(condition => condition.field === field.handle).length === 0)
```

Comment:
Repeat fields are filtered out correct, but now you've lost the ability to filter itself out.

e.g. if you're editing the `content` field, you shouldnt be able to pick `content`.